### DISCUSSION TITLE
Adguard home has not been completely removed

### DISCUSSION THREAD

**Discussion by moniga9** [AdguardTeam/AdGuardHome#2987] [Q&A]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. 

- [ x] I am running the latest version
- [x ] I checked the documentation and found no answer
- [x ] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  *  
* **How did you install AdGuard Home:**
  *  Snap
* **How did you setup DNS configuration:**
  *  Router
* **If it's a router or IoT, please write device model:**
  *  A3002RU
* **CPU architecture:**
  *  AMD64
* **Operating system and version:**
  *  Ubuntu 20.04.2

### Expected Behavior
 Complete removal of Adguard home from Ubuntu.

### Actual Behavior
 The setup webpage still works after removing Adguard Home snap. What should I do to remove it? 

### Screenshots
 

<details><summary>Screenshot:</summary>

 

![Screenshot_2021-04-19 Setup AdGuard Home](https://user-images.githubusercontent.com/45391043/115292141-e28b3c80-a155-11eb-8c8a-731169f4efa9.png)

</details>

**Comment by ameshkov** [comment]
1. Run `snap list` to make sure that `adguard-home` is not listed there
2. `snap remove adguard-home` to remove it
3. `ps aux | grep AdGuardHome` to check that there's no AdGuard Home process anymore

**Reply by moniga9** [reply]
Hello, please see. I've already deleted the snap package. But something like this came up after the last command.

```
root        1336  0.0  0.2 726212 42152 ?        SLsl kwi19   0:06 /opt/AdGuardHome/AdGuardHome -s run
lgawe      32285  0.0  0.0  11584  2652 pts/2    S+   01:04   0:00 grep --color=auto AdGuardHome
```

**Reply by ameshkov** [reply]
It seems you've installed it without snap.

Just stop it `/opt/AdGuardHome/AdGuardHome -s stop` and remove `/opt/AdGuardHome`

**Reply by moniga9** [reply]
I was able to remove the directory /opt/adhuardhome, but there is still something like this when I type this command.
ps aux | grep AdGuardHome

`lgawe 29 192 0.0 0.0 11584 736 pts / 2 S + 17:33 0:00 grep --color = auto AdGuardHome`

**Reply by ameshkov** [reply]
Looks like your own grep command.
